app/views/about/hiring.html.haml

```diff
@@ -0,0 +1,86 @@
+= render ViewComponents::AboutNav.new(:hiring)
+
+#page-hiring
+  %header
+    .lg-container.c-shapes
+      = graphical_icon "exercism-face"
+      %h1.text-h1.mb-12 Exercism is hiring!
+      %p.text-p-2xlarge Come join a passionate team making programming education fun and effective. Here are some of the roles we’re hiring for.
+
+  .roles.sm-container
+    = link_to hiring_1_about_path, class: 'role' do
+      = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Head of Partnerships and Fundraising
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Can you build and execute a fundraising strategy?
+          We're looking for someone to develop sponsorship and partnership revenue to support our growth as we strive to become the leading resource for lifelong programming education.
+        .details
+          .salary Salary: £50k (~$68k, ~€60k).
+          .remote Remote
+
+    = link_to hiring_2_about_path, class: 'role' do
+      = graphical_icon "hiring-content", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Content Factotum
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Passionate about programming education?
+          We're looking for someone to lead the content side of our platform, working with our community to create world-class resources. A varied and interesting role that mixes strategy, authoring and editing.
+        .details
+          .salary Salary: £50k (~$68k, ~€60k).
+          .remote Remote
+
+    = link_to hiring_3_about_path, class: 'role' do
+      = graphical_icon "hiring-community", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Community Manager
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Passionate about community?
+          Love chatting to people?
+          Interesting in creating systems that foster great discussions and connections?
+          Then you might be the perfect person to become our first full-time Community Manager!
+        .details
+          .salary Salary: £50k (~$68k, ~€60k).
+          .remote Remote
+
+    = link_to hiring_4_about_path, class: 'role' do
+      = graphical_icon "hiring-front-end-developer", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Front-end Developer
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Excited to build awesome UIs?
+          Love React and CSS?
+          Want to develop a technically interesting application used by millions of other developers?
```

Comment:
The RoR developer section has "build out" instead of "develop." Was that intentional? Develop to me sounds more like a greenfield assignment, whereas build out implies working with what is already there.

```suggestion
          Want to build out a technically interesting application used by millions of other developers?
```